templates/documentation/runbooks/pod-memory.md.tpl

```diff
@@ -10,12 +10,60 @@
 
 # {{ camelcase .Config.Name }} Pod Memory.\<type\> > 80% of limit last 30m
 
+Memory high alerts will occur if the memory rate is too high over x amount of time. The net impact is that customers might start seeing failures in {{ camelcase .Config.Name }} requests. On the other hand, none of the above might actually happen and the memory usage might drop down on it’s own. If the High Memory usage keeps repeating for the same host, there might be something wrong with the host itself and it might need recycling - all of this is described in much more detail below.
```

Comment:
Nit:
```suggestion
Memory high alerts will occur if the memory rate is too high over x amount of time. The net impact is that customers might start seeing failures in {{ camelcase .Config.Name }} requests. On the other hand, none of the above might actually happen and the memory usage might drop down on its own. If the High Memory usage keeps repeating for the same host, there might be something wrong with the host itself and it might need recycling - all of this is described in much more detail below.
```